packages/lesswrong/components/common/SectionFooter.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { PureComponent } from 'react';
+import { registerComponent } from 'meteor/vulcan:core';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+
+const styles = (theme) => ({
+  root: {
+    display: "flex",
+    justifyContent: "flex-end",
+    alignItems: "center",
+    marginTop: theme.spacing.unit*1.5,
+    marginBottom: theme.spacing.unit,
+    marginRight: theme.spacing.unit*3.5,
+    marginLeft: theme.spacing.unit,
+    color: theme.palette.text.secondary,
+    [theme.breakpoints.down('xs')]: {
+      flexWrap: "wrap"
+    },
+    '& > *': {
+      height: 30,
+      '&:after': {
+        content: '"•"',
+        marginLeft: theme.spacing.unit*2,
+        marginRight: theme.spacing.unit*2,
+      },
+      '&:last-child': {
+        '&:after': {
+          content: '""',
+          margin:0,
+        }
+      },
+      [theme.breakpoints.down('xs')]: {
```

Comment:
[Jim via Ray's keyboard] Hoist this above the `& > *` and invert it to `up('sm')`